README.md

```diff
@@ -59,66 +59,72 @@ More information about these configurations can be found in the [Deploy Sematic
+| `deployment.socket_io.dedicated`                          | Starts a separate dedicated Sematic server for Socket I/O             | `false`                                 |
+| `deployment.tolerations`                                  | K8S tolerations for Sematic server pods                               | `{}`                                    |
+| `deployment.worker_count`                                 | Number of WSGI workers per Sematic server pod (API only) `deployment.socket_io.dedicated` must be set to true if this is greater than 1 | `1`                                     |
+| `image.pull_policy`                                       | Image pull policy for Sematic server container images                 | `IfNotPresent`                          |
+| `image.pull_secrets`                                      | Image pull secrets for Sematic server container images                | Not set                                 |
+| `image.repository`                                        | Image repository for Sematic server container images                  | `sematicai/sematic-server`              |
+| `image.tag`                                               | Image tag for Sematic server container images                         | Defaults to version released in chart   |
+| `ingress.annotations`                                     | K8S ingress annotations for Sematic server                            | `{}`                                    |
+| `ingress.class_name`                                      | K8S ingress class name for Sematic server                             | `""`                                    |
+| `ingress.create`                                          | Enables creation of K8S ingress for Sematic server                    | `false`                                 |
+| `ingress.force_ssl`                                       | Forces redirects to HTTPS for Sematic Server even when HTTP is used   | `true`                                  |
+| `ingress.hosts`                                           | K8S ingress hosts YAML spec for Sematic server                        | See default values.yml                  |
+| `ingress.sematic_dashboard_url`                           | Public dashboard URL to K8S ingress for Sematic server                | Not set                                 |
+| `ingress.tls`                                             | K8s ingress TLS YAML spec for Sematic server                          | `[]`                                    |
+| `local_storage.enabled`                                   | Enables using local disk storage for Artifacts for the Sematic server | `false`                                 |
+| `local_Storage.local_storage_path`                        | Sets the local storage path to use for Artifacts                      | `~/.sematic/data`                       |
```

Comment:
```suggestion
| `local_storage.path`                        | Sets the local storage path to use for Artifacts                      | `~/.sematic/data`                       |
```

fixed wrong capitalization and removed redundant `local_storage_`

Reply:
Fixed the capitalization. The config name is already released like this, so keeping the duplicated name.